templates/leaf/deploymentTopLayer.json

```diff
@@ -0,0 +1,83 @@
+{
+    "modulesContent": {
+        "$edgeAgent": {
+            "properties.desired": {
+                "modules": {
+                    "registry": {
+                        "settings": {
+                            "image": "registry:latest",
+                            "createOptions": "{\"HostConfig\":{\"PortBindings\":{\"5000/tcp\":[{\"HostPort\":\"5000\"}]}}}"
+                        },
+                        "type": "docker",
+                        "version": "1.0",
+                        "env": {
+                            "REGISTRY_PROXY_REMOTEURL": {
+                                "value": "https://mcr.microsoft.com"
+                            } 
+                        },
+                        "status": "running",
+                        "restartPolicy": "always"
+                    },
+                    "IoTEdgeAPIProxy": {
+                        "settings": {
+                            "image": "mcr.microsoft.com/azureiotedge-api-proxy:1.0",
+                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"8000/tcp\": [{\"HostPort\":\"8000\"}]}}}"
```

Comment:
Change to 443 (see this - https://github.com/Azure-Samples/iotedge_config_cli/pull/17)